open_evse.ino

```diff
@@ -441,18 +435,23 @@ void OnboardDisplay::LcdMsg_P(PGM_P l1,PGM_P l2)
   LcdPrint_P(1,l2);
 }
 
-
 // print at (0,y), filling out the line with trailing spaces
 void OnboardDisplay::LcdPrint(int y,const char *s)
 {
-  m_Lcd.setCursor(0,y);
+  LcdPrint(0,y,s);
+}
+// print at (x,y), filling out the line with trailing spaces
+void OnboardDisplay::LcdPrint(int x, int y,const char *s)
+{
+  m_Lcd.setCursor(x,y);
   uint8_t i,len = strlen(s);
-  if (len > LCD_MAX_CHARS_PER_LINE)
-    len = LCD_MAX_CHARS_PER_LINE;
+  int max = LCD_MAX_CHARS_PER_LINE - x;
+  if (len > max)
+    len = max;
   for (i=0;i < len;i++) {
     m_Lcd.write(s[i]);
   }
-  for (i=len;i < LCD_MAX_CHARS_PER_LINE;i++) {
+  for (i=len;i < max;i++) {
```

Comment:
This will always clear to the end of the line.  What if I wanted to put a single character in a specific location on the display ?